### PR TITLE
feat: added description for missing resources

### PR DESCRIPTION
**Pull request by mario-mazo**
# Contributor checklist

- [ ] Bug fixes include regression tests
- [x] Features include unit/acceptance tests

### CODE REVIEW

**Comment by mario-mazo** [comment]
add describe to missing resources, should close #58

**Comment by mario-mazo** [comment]
the structs I did not spec is bc some fields were not in the `@schema` and I have no idea what type are those

**Comment by zachdaniel** [comment]
@mario-mazo great stuff! I've added some basic tests for each of these entities in master, so you can add a test to confirm that the description is properly added when you add a description. Let me know if you need help with that.

File: lib/ash/resource/aggregate.ex [hunk 1]
```diff
@@ -22,10 +22,20 @@ defmodule Ash.Resource.Aggregate do
...
+      doc: "An optional description for the aggregate"
     ]
   ]
 
-  @type t :: %__MODULE__{}
+  @type t :: %__MODULE__{
+          name: atom(),
+          relationship_path: {:ok, list(atom())} | {:error, String.t()},
+          filter: Keyword.t(),
+          kind: [:count],
```
Comment (zachdaniel) [inline lib/ash/resource/aggregate.ex:36]: This will just be `kind: :count`

File: lib/ash/resource/validation.ex [hunk 2]
```diff
@@ -39,9 +39,17 @@ defmodule Ash.Resource.Validation do
...
   """
-  defstruct [:validation, :module, :opts, :expensive?, on: []]
+  defstruct [:validation, :module, :opts, :expensive?, :description, on: []]
+
+  @type t :: %__MODULE__{
+          validation: {atom(), list(atom())},
+          module: atom(),
+          opts: list(atom()),
+          expensive?: boolean(),
+          description: String.t(),
```
Comment (zachdaniel) [inline lib/ash/resource/validation.ex:49]: `String.t() | nil`

File: lib/ash/resource/identity.ex [hunk 3]
```diff
@@ -14,12 +14,20 @@ defmodule Ash.Resource.Identity do
...
     ]
   ]
 
   def schema, do: @schema
 
-  @type t :: %__MODULE__{}
+  @type t :: %__MODULE__{
+          name: atom(),
+          keys: list(atom()),
+          description: String.t()
```
Comment (zachdaniel) [inline lib/ash/resource/identity.ex:29]: `String.t() | nil`

File: lib/ash/resource/calculation/calculation.ex [hunk 4]
```diff
@@ -12,10 +12,19 @@ defmodule Ash.Resource.Calculation do
...
+      doc: "An optional description for the calculation"
     ]
   ]
 
-  @type t :: %__MODULE__{}
+  @type t :: %__MODULE__{
+          name: atom(),
+          calculation: {:ok, {atom(), any()}} | {:error, String.t()},
+          arguments: list(any()),
+          description: String.t()
```
Comment (zachdaniel) [inline lib/ash/resource/calculation/calculation.ex:26]: `String.t() | nil`

**Review by zachdaniel — APPROVED** [review]
Great work!
🚀 Thank you for your contribution! 🚀

**Pull request merged**